app-frontend/src/app/components/toolSearch/toolSearch.html

```diff
@@ -13,9 +13,10 @@
                  id="search-input"
                  class="form-control input-dark"
                  placeholder="Search for tools"
-                 ng-model="$ctrl.searchText"/>
-          <button type="button" class="btn btn-link">
-            <span class="close">x</span>
+                 ng-model="$ctrl.searchText"
+                 ng-keyup="$event.keyCode == 13 && $ctrl.onSearchAction()"/>
```

Comment:
Handling `enter` key press. There is no submit button, so this method is necessary to initiate a search.